### ISSUE TITLE
Release

### ISSUE THREAD

**Issue by t0m5k1** [Miserlou/RJModules#1]
Please can we have a release for Windows, Linux, MAC?

**Comment by phdsg** [comment]
[RJModules-deve3dd329-lin.zip](https://github.com/Miserlou/RJModules/files/1503878/RJModules-deve3dd329-lin.zip)

**Comment by t0m5k1** [comment]
Thank you so much :)

**Comment by Miserlou** [comment]
Hey Tom and PhDSG - no official release left as I still have a few more modules I want to make today. Hopefully I'll be able to push something out this evening, but I don't have a Windows machine to build on..

**Comment by nay-seven** [comment]
hello , looks great but still no Mac releases ? ;-)

**Comment by Miserlou** [comment]
I have.. One! (Maybe two?) More modules before I do a release! It's nearly done!

**Comment by nay-seven** [comment]
ok, no worries, finally found one on FB group to start testing ;-)

**Comment by Miserlou** [comment]
..do you have a link to that?

**Comment by a1laserboy** [comment]
[RJModules.zip](https://github.com/Miserlou/RJModules/files/1522801/RJModules.zip)

I don't know if this will work, first time trying to upload something here. I can't find the original fb post though, or even remember who built it.

**Comment by nay-seven** [comment]
yes, was done by Jeremy Wentworth on official group, looks it's here , on this github ;-) 
[https://github.com/Miserlou/RJModules/files/1511708/RJModules.zip](url)

**Comment by Atavic** [comment]
Subscribed, I want the releases.

**Comment by jeremywen** [comment]
Are these your official releases?  Are you going to put these on the release page?  I am trying to get these plugin manifests updated.  https://docs.google.com/spreadsheets/d/19EuENTzFQTlhYqIuCwJ_aZbEJ3RTIv3ZzqghNLK6p04/edit#gid=0

**Comment by Miserlou** [comment]
Okay, I'm definitely overdue for a release, but I need to be able to compile and sign my own archives for each of the target architectures.

I'm thinking about using https://github.com/dockcross/dockcross for this - does anybody have any Dockcross experience?

**Comment by jeremywen** [comment]
Try it.

**Comment by Miserlou** [comment]
Okay, so my experiment with Docker doesn't work (yet?) because OSX uses clang and Linux use gcc - documented here: https://github.com/VCVRack/Rack/pull/509

I'd like to keep making more modules, but I told myself I wouldn't make any more until I made a release for what has already been made.

Can somebody please make a Windows build from HEAD that I can publish as version 0 so that I can get back to making modules? :) Linux also welcome.

(Also if you guys aren't playing with Twin LFO and Sidechain yet, check them out, super fun!)

**Comment by phdsg** [comment]
[RJModulesWIN.0.5.0.zip](https://github.com/Miserlou/RJModules/files/1560042/RJModulesWIN.0.5.0.zip)

**Comment by phdsg** [comment]
this should work...
[RJModules-0.5.0-win.zip](https://github.com/Miserlou/RJModules/files/1560068/RJModules-0.5.0-win.zip)

**Comment by Miserlou** [comment]
Nice, thanks!

**Comment by Atavic** [comment]
Great work everyone !! :+1:

**Comment by nay-seven** [comment]
Congrats for this first official one and great idea for the prefix! 
thanks!
